features/test/src/static/manifest.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It _should_ be possible to leave the manifest out - might make sense for this one.

Reply:
Just delete it?